packages/services/f-feature-management/test/featureGetter.test.js

```diff
@@ -0,0 +1,118 @@
+import { getFeature, init, poll } from '../src/featureGetter';
+import fetchMock from "jest-fetch-mock";
+
+fetchMock.enableMocks();
+
+const settings = {
+  scope: 'test-scope',
+  environment: 'test-env',
+  key: 'test-key'
+};
+
+const fetchResponse = {
+  createdAt: '2021-09-10 15:00',
+  features: [
+    {
+      key: 'key1',
+      testVal: 'val1'
+    }
+  ]
+};
+
+const flushPromises = () => new Promise(res => process.nextTick(res))
+
+describe('When calling poll', () => {  
+
+  beforeEach(async () => {
+    fetch.resetMocks();
+    jest.useFakeTimers();
+
+    fetch.mockResponse(JSON.stringify(fetchResponse));
+
+    await poll(settings);    
+  });
+
+  it('Makes an initial call to fetch', async () => {
+    
+    expect(fetch).toHaveBeenCalledTimes(1);
+    expect(fetch).toHaveBeenCalledWith(`https://features.api.justeattakeaway.com/config/v1/${settings.scope}/${settings.environment}-${settings.key}`);
+  });
+
+  it('Starts polling', async () => {
```

Comment:
Test descriptions should also start with the word "should", just another convention of ours, e.g.:
```suggestion
  it('should start polling', async () => {
```